packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -26,9 +26,12 @@ spec:
           volumeMounts:
             - mountPath: /fluent-bit/etc/
               name: config
-            - mountPath: /run/log/journal
+            - mountPath: {{if .Values.global.seLinux.enabled }}"/run/log/journal"{{ else }}"/var/log/journal"{{ end }}
```

Comment:
It doesn't have anything to do with selinux, it's just the way systemd is configured by default for the distro.

On CentOS:

```
$ grep Storage /etc/systemd/journald.conf
#Storage=auto
```

On Ubuntu:

```
$ grep Storage /etc/systemd/journald.conf 
Storage=persistent
```